app/controllers/trains_controller.rb

```diff
@@ -24,6 +24,7 @@ def update
   end
 
   def destroy
+    response = TrainApi.new.destroy(params[:id])
+    redirect_to trains_path
```

Comment:
i think we should make sure some kind of error handling exists on this incase the api ever responds with an invalid response.
also i think we should also have a message for when this action is successful in our app.